### DISCUSSION TITLE
Question about BladeDocumentTemplate Path

### DISCUSSION THREAD

**Discussion by jeffdodge** [creagia/laravel-sign-pad#7] [Q&A]
So I have got everything to work except for the blade template, it keeps telling me that the file is not found that I created as the template. 

InvalidArgumentException (PHP 8.1.8 / 9.38.0)
View [.Users.jeff.Dev.sigPDF.resources.pdf.sigTemplate.blade.php] not found.

I've tried using resource_path('pdfs/sigTemplate.blade.php') without any luck.    When I use just pdfs/... It says it's not found too.

I guess my question is, what directory do I put the blade template file into?   I thought it would of been in resources/pdf/ 

Thanks for your help.

**Comment by dtorras** [comment]
I've reviewed the docs, and it's probably not clear enough. The path for the Blade template should be inside the `resources/views` directory. Just like you use the `view()` method. For example, `view('path/file.blade.php')` references to `resources/views/path/file.blade.php`.
